pkg/telemetry/client.go

```diff
@@ -16,7 +16,7 @@ type Client struct {
 	dispatcher *event.Dispatcher
 }
 
-const localhost = "127.0.0.1"
+const localhost = "0.0.0.0"
```

Comment:
is there a specific reason why we're doing 0.0.0.0 here

Reply:
good question 😄  to be honest, I tried running it in either `docker-compose` or as a `binary` but I kept having issues in receiving the packages from the PS5 to my laptop (same network of course). The only way I could make it work was to change the `localhost` address. It may be a problem with my router. I can revert back or perhaps this can be passed as flag or something. What do you think?

Reply:
i think we should write a document about it 😄: https://github.com/anilmisirlioglu/f1-telemetry-go/blob/master/pkg/telemetry/client.go#L30-L45

let's keep it as 127.0.0.1 for now, I'll ask a few more people about it and try to find out why

Reply:
done 😄

Reply:
thanks 😄 I will come up with a create a new tag for f1-2020 support and merge this pr for f1-2021, I haven't tried this pr, but I guess it works

Reply:
that's a good idea 😄  You can trust the classic `it works on my computer` 🤣